app/institutions/dashboard/-components/institutional-users-list/component.ts

```diff
@@ -0,0 +1,57 @@
+import { action, computed } from '@ember-decorators/object';
+import { alias } from '@ember-decorators/object/computed';
+import Component from '@ember/component';
+import translations from 'ember-osf-web/locales/en/translations';
+import InstitutionModel from 'ember-osf-web/models/institution';
+
+export default class InstitutionalUsersList extends Component {
+    @alias('model.taskInstance.value') institution!: InstitutionModel;
+
+    // Private properties
+    defaultDepartment = translations.institutions.dashboard.select_default;
+    department?: string;
+    sort = '-userFullName';
+
+    @computed('institution')
+    get departments() {
+        let departments = [this.defaultDepartment];
```

Comment:
This should instead use the `t` method from the `i18n` service, e.g.
https://github.com/CenterForOpenScience/ember-osf-web/blob/a9756531e95ddffdede1e37c7717408da9eae426/lib/osf-components/addon/components/delete-button/component.ts#L43

Reply:
Should we use the t method in these other methods? this.defaultDepartment is set when the component is created with the t method but should never change. this.department is used to keep track of the selected department, but this.defaultDepartment if just used for reference to know if the "All Departments" option is selected.

Reply:
The problem here is that you've imported the english translation instead of calculating it based on current locale. I don't think you can use the `t` method to initialize `defaultDepartment` because I'm pretty sure service injections are not available for class field initialization (not 100% sure on this, so you might want to try it). You *could* make `defaultDepartment` a computed that returns `i18n.t('institutions.dashboard.select_default')` instead of calling the `t` method each time. The computed would be dependent on `i18n.locale`.